py/kubeflow/testing/create_kf_instance.py

```diff
@@ -62,6 +62,14 @@ def main(): # pylint: disable=too-many-locals,too-many-statements
     "--cluster_num",
     default="", type=int, help=("Number of cluster to deploy to."))
 
+  parser.add_argument(
```

Comment:
Why are you passing this as an argument? Why don't you just have the code compute the timestamp?

Reply:
this timestamp is the time deployed codes taken snapshot, not the time it is deployed.